manifests/router/link_route_pattern.pp

```diff
@@ -10,7 +10,7 @@
 #
 define qpid::router::link_route_pattern(
   $prefix    = 'queue.',
-  $connector = 'broker',
+  $connector = '',
```

Comment:
`undef` is unreliable across puppet versions.  I could use UNSET, but the rest of the module uses empty strings.